packages/fxa-auth-server/lib/metrics/events.js

```diff
@@ -140,6 +140,10 @@ module.exports = (log, config) => {
         });
         await amplitude('flow.complete', request, data, metricsContext);
 
+        if (metricsContext.flowType === 'login') {
+          glean.login.complete(request, { uid: data?.uid ?? '' });
+        }
```

Comment:
The signal's registered at https://github.com/mozilla/fxa/blob/1386b2de515ddc0a078083ff1d7e3ebde2f05484/packages/fxa-auth-server/lib/routes/utils/signin.js#L228-L241 (which include some relevant comments).